CONTRIBUTING.md

```diff
@@ -1,331 +1,444 @@
 
-  // external packages
-  "github.com/foo/bar"
-  "gopkg.io/baz.v1"
-)
-```
+<https://github.com/go-gitea/gitea/pulls?q=is%3Apr+label%3Areviewed%2Fwait-merge+sort%3Acreated-asc+is%3Aopen>
 
-## Design guideline
+Gitea uses it's own tool, the <https://github.com/GiteaBot/gitea-backporter> to automate parts of the review process. \
+This tool does the things listed below automatically:
+- create a backport PR if needed once the initial PR was merged
+- remove the PR from the merge queue after the PR merged
+- keep the oldest branch in the merge queue up to date with merges
 
-To maintain understandable code and avoid circular dependencies it is important to have a good structure of the code. The Gitea code is divided into the following parts:
-
-- **models:** Contains the data structures used by xorm to construct database tables. It also contains supporting functions to query and update the database. Dependencies to other code in Gitea should be avoided although some modules might be needed (for example for logging).
-- **models/fixtures:** Sample model data used in integration tests.
-- **models/migrations:** Handling of database migrations between versions. PRs that changes a database structure shall also have a migration step.
-- **modules:** Different modules to handle specific functionality in Gitea. Shall only depend on other modules but not other packages (models, services).
-- **public:** Frontend files (javascript, images, css, etc.)
-- **routers:** Handling of server requests. As it uses other Gitea packages to serve the request, other packages (models, modules or services) shall not depend on routers.
-- **services:** Support functions for common routing operations. Uses models and modules to handle the request.
-- **templates:** Golang templates for generating the html output.
-- **tests/e2e:** End to end tests
-- **tests/integration:** Integration tests
-- **tests/gitea-repositories-meta:** Sample repos used in integration tests. Adding a new repo requires editing `models/fixtures/repositories.yml` and `models/fixtures/repo_unit.yml` to match.
-- **tests/gitea-lfs-meta:** Sample LFS objects used in integration tests. Adding a new object requires editing `models/fixtures/lfs_meta_object.yml` to match.
-- **vendor:** External code that Gitea depends on.
+### Final call
 
-## Documentation
+If a PR has been ignored for more than 7 days with no comments or reviews, and the author or any maintainer believes it will not survive a long wait (such as a refactoring PR), they can send "one last call" to the TOC by mentioning them in a comment.
 
-If you add a new feature or change an existing aspect of Gitea, the documentation for that feature must be created or updated.
+After another 7 days, if there is still zero approval, this is considered a polite refusal, and the PR will be closed to avoid wasting further time. Therefore, the "final call" has a cost, and should be used cautiously.
```

Comment:
I'm not sure if I like the wording here. 
Saying the final call has a cost whereby potentially a PR will simply be closed doesn't sound entirely friendly.

I think in most cases a response of some form will be given, rather than a quiet closing. If that's not the case, it should be the case imo

Reply:
I think once a final call has been sent, the maintainers should comment/review ASAP to avoid a quiet closing.

So whether it is friendly depends on how the maintainers respond to the final call. And if someone always sends final calls for their unimportant PRs, I think "a quiet closing" is the most friendly way.